docs/framework/api-architecture.md

```diff
@@ -22,10 +22,10 @@ Usage documentation is in the [APIv3](/api/index.md) page.
 -   API functions should match the relevant BAO names & have the
     functions 'get', 'create' and delete'
 -   All functions should receive $params as an array(not a reference)
--   All functions should return either
-    -   an id indexed array of results via
+-   All functions should either
+    -   return an id indexed array of results via
         the `civicrm_api3_create_success()` function
-    -   an appropriate message via throw new api_Exception
```

Comment:
@eileenmcnaughton should there be anything about sequential or does that get dealt with by civicrm_api3_create_success() ?